com.transmissionbt.Transmission.json

```diff
@@ -47,6 +47,13 @@
         },
         {
             "name": "transmission",
+            "cleanup": [
+              "/bin/transmission-create",
+              "/bin/transmission-daemon",
+              "/bin/transmission-edit",
+              "/bin/transmission-show",
+              "/share/transmission"
```

Comment:
We should not remove this data. To see why, go to Preferences → Remote, tick “Allow remote access”, and then click “Open web client”.

Reply:
Oh interesting, so `transmission-gtk` has the daemon built in basically?

Reply:
Both `transmission-gtk` and `transmission-daemon` instantiate a `tr_session` from libtransmission (an internal static library) which contains a `tr_rpc_server`.